internal/provider/resource_firewall_rule.go

```diff
@@ -169,6 +169,51 @@ func resourceFirewallRule() *schema.Resource {
 				Optional:     true,
 				ValidateFunc: validation.StringInSlice([]string{"match-ipsec", "match-none"}, false),
 			},
+			"weekdays": {
+				Description: "Days of the week the rule is enforced.",
+				Type:        schema.TypeString,
```

Comment:
This should probably be a set, similar to `src_firewall_group_ids`:
```suggestion
				Type:        schema.TypeSet,
				Elem: &schema.Schema{Type: schema.TypeString},
```